modules/apps/frontend-js/frontend-js-spa-web/package.json

```diff
@@ -2,6 +2,7 @@
 	"dependencies": {
 		"frontend-js-web": "*"
 	},
+	"main": "init.js",
```

Comment:
Just to confirm my understanding is correct here...

Rather than adding `"main": "init.js"` we could've simply renamed it to `index.js`?

Reply:
Yes, though it is not a safe refactor, because someone may be invoking it through AMD loader by name. I know it's not very probable, but if you want to make that change and be safe, you must make sure of that.